src/DssDirectDepositHub.sol

```diff
@@ -16,6 +16,12 @@
 
 pragma solidity 0.6.12;
 
+import { D3MPoolInterface } from "./pools/D3MPoolInterface.sol";
+import { D3MPlanInterface } from "./plans/D3MPlanInterface.sol";
+
+interface D3MPoolLike is D3MPoolInterface {}
```

Comment:
Hmmm I have my doubts about it, in general we prefer to have the interfaces described with the very minimum in the same contract. It's true that in this case the Interface file defines all the ones being used in the Hub, so we could make an exception. In that case I think I'd just prefer to use `D3MPoolInterface` instead of `D3MPoolLike`

Reply:
you're totally correct :-D I'll fix it